lizard_auth_server/models.py

```diff
@@ -450,3 +450,12 @@ def __unicode__(self):
         else:
             return "{role} in {org}".format(
                 role=self.role, org=self.organisation)
+
+    def as_dict(self):
+        return {
+            "portal_name": self.role.portal.name,
+            "organisation_name": self.organisation.name,
+            "organisation_uuid": self.organisation.unique_id,
+            "role_name": self.role.name,
+            "role_code": self.role.code,
+            "role_uuid": self.role.unique_id}
```

Comment:
Eventueel kan hier ook

    return {
          "organisation": self.organisation.as_dict(),
          "role": self.role.as_dict()
     }

(portal hoeft in principe niet, want client code weet zelf altijd wel welke portal hij is)